Assets/HoloToolkit/Input/Scripts/Utilities/MotionControllerVisualizer.cs

```diff
@@ -42,6 +42,7 @@ public class MotionControllerVisualizer : MonoBehaviour
 
         // This will be used to keep track of our controllers, indexed by their unique source ID.
         private Dictionary<uint, MotionControllerInfo> controllerDictionary = new Dictionary<uint, MotionControllerInfo>(0);
+        private List<uint> loadingControllers = new List<uint>();
```

Comment:
Could we use the `controllerDictionary` instead of creating a new list?

Reply:
Oh, the description of why is in the title of your PR. 😆

Reply:
I thought about that, perhaps adding the ID and a null object to the Dictionary, but I thought that might cause issues somewhere along the line. If I add enough null checks as well though, that might work.